src/app/lib/utilities/preprocessor/rules/cpstypes.js

```diff
@@ -0,0 +1,27 @@
+import deepGet from '../../../../helpers/json/deepGet';
+
+const whitelist = ['STY', 'MAP'];
+
+const filterUnknownCpsTypes = data => {
+  const groups = deepGet(['content', 'groups'], data);
+
+  if (!groups) {
+    return data;
+  }
+
+  const newGroups = groups.map(g => {
+    const ng = g;
```

Comment:
Do we need this line?

Reply:
Not doing this causes complaints from <https://eslint.org/docs/rules/no-param-reassign>, as it was done in the [topstories preprocessor rule](https://github.com/bbc/simorgh/blob/85c8989f2370144b35e8a26ec73807e1d065d658/src/app/lib/utilities/preprocessor/rules/topstories.js#L11) (for as far as I can tell the same reason) I thought to keep the same pattern

Reply:
Ahhh thats what its for :P Gotcha, can you make the names words please :P